src/artemis/devices/synchrotron.py

```diff
@@ -0,0 +1,28 @@
+from ophyd import Component, Device, EpicsSignal
+
+
+class SynchrotoronMachineStatus(Device):
+    synchrotron_mode: EpicsSignal = Component(
+        EpicsSignal, "CS-CS-MSTAT-01:MODE", string=True
+    )
+    ring_energy: EpicsSignal = Component(EpicsSignal, "CS-CS-MSTAT-01:BEAMENERGY")
+    beam_dump_countdown: EpicsSignal = Component(
+        EpicsSignal, "CS-CS-MSTAT-01:USERCOUNTDN"
+    )
+
+
+class SynchrotronTopUp(Device):
+    topup_start_countdown: EpicsSignal = Component(
+        EpicsSignal, "SR-CS-FILL-01:COUNTDOWN"
+    )
+    topup_end_countdown: EpicsSignal = Component(
+        EpicsSignal, "SR-CS-FILL-01:ENDCOUNTDN"
+    )
+
+
+class Synchrotron(Device):
+
+    machine_status: SynchrotoronMachineStatus = Component(SynchrotoronMachineStatus, "")
+    top_up: SynchrotronTopUp = Component(SynchrotronTopUp, "")
```

Comment:
**should**: You can do the following to remove repetition elsewhere in the device.
```suggestion
    machine_status: SynchrotoronMachineStatus = Component(SynchrotoronMachineStatus, "CS-CS-MSTAT-01:")
    top_up: SynchrotronTopUp = Component(SynchrotronTopUp, "SR-CS-FILL-01:")
```